tasks/functions/functions.js

```diff
@@ -15,31 +15,44 @@ function sum(a, b) {
  * }
  */
 function getFullName(object) {
-
+	return object.firstName + " " + object.lastName;
```

Comment:
JFYI

Also for concatenation, you could use the template strings (https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Template_literals)